prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -33,20 +33,22 @@
 #ifndef _VERBS_RDM_H
 #define _VERBS_RDM_H
 
+#include <rdma/rdma_cma.h>
+#include "../uthash.h"
+
 #include "../fi_verbs.h"
 #include "verbs_utils.h"
-#include "../uthash.h"
 #include "verbs_tagged_ep_rdm_states.h"
 
-
-#define FI_IBV_RDM_CONN_SELF ((struct fi_ibv_rdm_tagged_conn *) 0x1)
-
 #define FI_IBV_RDM_ADDR_STR_FORMAT "[%02x:%02x:%02x:%02x:%02x:%02x]"
 
-#define FI_IBV_RDM_ADDR_STR(addr)				   \
-        *((unsigned char*) addr), *((unsigned char*) addr + 1),	   \
-        *((unsigned char*) addr + 2),*((unsigned char*) addr + 3), \
-        *((unsigned char*) addr + 4),*((unsigned char*) addr + 5)
+#define FI_IBV_RDM_ADDR_STR(addr)				\
+        *((unsigned char*) addr + sizeof(sa_family_t) + 0),	\
+	*((unsigned char*) addr + sizeof(sa_family_t) + 1),	\
+	*((unsigned char*) addr + sizeof(sa_family_t) + 2),	\
+	*((unsigned char*) addr + sizeof(sa_family_t) + 3),	\
+	*((unsigned char*) addr + sizeof(sa_family_t) + 4),	\
+	*((unsigned char*) addr + sizeof(sa_family_t) + 5)
```

Comment:
This is kind of ugly.  I think we need a common call for converting a sockaddr into a string -- one that uses inet_ntoa.  There may be one in the code already somewhere.